test/handler.test.js

```diff
@@ -199,9 +200,31 @@ describe('strong-error-handler', function() {
       requestJson().end(function(err, res) {
         if (err) return done(err);
 
-        var expectedData = {statusCode: 500, stack: error.stack};
-        for (var key in error) expectedData[key] = error[key];
+        var expectedData = {
+          statusCode: 500,
+          message: 'a test error message',
+          name: 'ErrorWithProps',
+          details: 'some details',
+          extra: 'sensitive data',
+          stack: error.stack,
+        };
+        expect(res.body).to.have.property('error');
+        expect(res.body.error).to.eql(expectedData);
+        done();
+      });
+    });
 
+    it('contains name property when debug=true', function(done) {
```

Comment:
I see, that makes a lot of sense. In that case I am proposing to rework this test a bit:

 - rename to "contains non-enumerable Error properties when debug=true"
 - check only for message, name and stack

This should make it more clear how is this test different from the test above.